jupyterhub/schema.yaml

```diff
@@ -2220,6 +2216,37 @@ properties:
         type: [integer, "null"]
         description: See the `--max-age` flag.
 
+  defaultTolerations:
+    type: object
+    required: [enabled]
+    description: |
+      Default [Toleration objects](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#toleration-v1-core)
+      applied to all Pods.  If your cluster has a [PodTolerationRestriction 
+      admission-controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction)
+      then you may need to configure it to accept these tolerations, 
+      or disable the defaultTolerations.
+    properties:
+      enabled:
+        type: boolean
+        description: Enable/disable default tolerations (enabled by default)
+      tolerations:
+        type: array
+        description: |
+          ```
+          - key: hub.jupyter.org/dedicated
+            operator: Equal
+            value: user
+            effect: NoSchedule 
+          - key: hub.jupyter.org_dedicated
+            operator: Equal
+            value: user
+            effect: NoSchedule
+          ```
```

Comment:
I suggest we split this in two, one named userTolerations and one named coreTolerations.